pkg/test/integration/server.go

```diff
@@ -149,33 +164,54 @@ func New(cfg *rest.Config, o ...Option) (*Manager, error) {
 		op(c)
 	}
 
+	// Create temporary directory for CRDs.
+	dir, err := ioutil.TempDir(".", "")
+	if err != nil {
+		return nil, errors.Wrap(err, errCreateTmpDir)
+	}
+
+	// Download CRDs into subdirectories of temp.
+	crdPaths := []string{}
+	for _, path := range c.CRDPaths {
+		dst, err := downloadPath(path, dir)
+		if err != nil {
+			return nil, errors.Wrap(err, errGetCRDs)
+		}
+		crdPaths = append(crdPaths, dst)
+	}
+
+	// Create environment configuration.
 	e := &envtest.Environment{
-		CRDDirectoryPaths:  c.CRDDirectoryPaths,
+		CRDDirectoryPaths:  crdPaths,
 		Config:             cfg,
 		UseExistingCluster: &useExisting,
 	}
 
-	cfg, err := e.Start()
+	// Start environment.
```

Comment:
Nitpick: Do you think the comments in this `New` function are adding value? Many people (including myself) feel comments are best when they explain _why_ something is happening, not _what_ is happening. In this case the comments appear to be explaining what is happening, and I'm not sure they really need to given how readable the code is.

Reply:
Nope! Happy to remove them.